sfepy/discrete/conditions.py

```diff
@@ -55,7 +57,16 @@ def from_conf(conf, regions):
         for key, cc in six.iteritems(conf):
             times = cc.get('times', None)
 
-            if 'ebc' in key:
+            if 'dgebc' in key:
```

Comment:
Just from the "new feature to the end" perspective, I would put the DG conditions after the FEM ones. That is, above 'lcbc'...

Reply:
Ok, should I put it before or after IC?

Reply:
I thought after epbc, before lcbc, but after ic would do as well.

Reply:
I have encoutered issue similiar to the class hierarchy problem in dof_info, if the condition is "dgebc" the `key` contains "ebc", so they can't swapped without changing the decision logic to something like
```python
if (not "dgebc" in key) and ("ebc" in key):
   ...
```

Reply:
:) True, good catch. What about `key.startswith()`?